pipelines/live-1/main/tzcronjobber.yaml

```diff
@@ -0,0 +1,73 @@
+resources:
+- name: tools-image
+  type: docker-image
+  source:
+    repository: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tools
+    tag: latest
+    aws_access_key_id: ((aws-live-1.access-key-id))
+    aws_secret_access_key: ((aws-live-1.secret-access-key))
+- name: cronjobber-image
+  type: docker-image
+  source:
+    repository: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tools
+    tag: cronjobber
+    aws_access_key_id: ((aws-live-1.access-key-id))
+    aws_secret_access_key: ((aws-live-1.secret-access-key))
+jobs:
+  - name: cronjobber
+    plan:
+      - get: tools-image
+      - task: download
+        image: tools-image
+        config:
+          platform: linux
+          params:
+            RELEASE: 0.2.0
```

Comment:
Instead of specifying the release number and downloading the archive with `wget`, we could take advantage of the [github-release-resource](https://github.com/concourse/github-release-resource) to automate builds on new releases.

Otherwise, building a new image means we'll have to update the spec here and manually trigger the job.

Reply:
release-resource broken, see https://github.com/concourse/github-release-resource/issues/56 had to revert to wget